src/test/java/com/github/awwkoala/selenium/DynamicControlsTests.java

```diff
@@ -0,0 +1,58 @@
+package com.github.awwkoala.selenium;
+
+import com.github.awwkoala.selenium.page.DynamicControlsPage;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class DynamicControlsTests extends TestBase {
+
+  private DynamicControlsPage dcPage;
```

Comment:
Since nearly everyone coding in java are using IDEs, and not coding by hand, it's more preferable to me to use full variable names, and not abbreviate it, so I would use in this example dynamicControlsPage. But it's fine too